src/psm/test/aes_test_multiple_bterms.ok

```diff
@@ -4,25 +4,23 @@
 [INFO ODB-0131]     Created 19835 components and 101835 component-terminals.
 [INFO ODB-0132]     Created 2 special nets and 39670 connections.
 [INFO ODB-0133]     Created 18908 nets and 62165 connections.
-[INFO PSM-0022] Using 1.100V for VDD
-[INFO PSM-0076] Setting metal node density to be standard cell height times 5.
-[INFO PSM-0031] Number of PDN nodes on net VDD = 43813.
-[INFO PSM-0064] Number of voltage sources = 2250.
-[INFO PSM-0040] All PDN stripes on net VDD are connected.
+[INFO PSM-0040] All shapes on net VDD are connected.
 ########## IR report #################
-Corner: default
+Corner           : default
+Supply voltage   : 1.10e+00 V
 Worstcase voltage: 1.10e+00 V
-Average IR drop  : 7.60e-05 V
-Worstcase IR drop: 3.97e-03 V
+Average voltage  : 1.10e+00 V
+Average IR drop  : 2.34e-04 V
+Worstcase IR drop: 2.50e-03 V
+Percentage drop  : 0.23 %
 ######################################
-[INFO PSM-0022] Using 0.000V for VSS
-[INFO PSM-0076] Setting metal node density to be standard cell height times 5.
-[INFO PSM-0031] Number of PDN nodes on net VSS = 42314.
-[INFO PSM-0064] Number of voltage sources = 2250.
-[INFO PSM-0040] All PDN stripes on net VSS are connected.
+[INFO PSM-0040] All shapes on net VSS are connected.
 ########## IR report #################
-Corner: default
-Worstcase voltage: 6.75e-03 V
-Average IR drop  : 1.14e-04 V
-Worstcase IR drop: 6.75e-03 V
+Corner           : default
+Supply voltage   : 0.00e+00 V
+Worstcase voltage: 4.25e-03 V
+Average voltage  : 3.49e-04 V
+Average IR drop  : 3.49e-04 V
+Worstcase IR drop: 4.25e-03 V
```

Comment:
Why are the voltage values equal to the IR drop values?  It would be helpful to print the net name in the report.

Reply:
I think adding the net name makes sense, I'll add it.